### ISSUE TITLE
Modal's "top" property makes it disappear on screen resize

### ISSUE THREAD

**Issue by sjmiller7** [codeRIT/brickhack.io#1153]
When the modal is opened, the JS calculates the top property as the current scrollpoint on the page + some extra for whitespace. 

When the window is resized while the modal is open, it appears to jump to a different part of the page (because the change in number of columns messes with page height somehow)

We can't just use fixed position because then that messes with the scrolling on mobile sized (that we have to have in order to show the full image without distortion or tiny sizing).

**Comment by colbygallup** [comment]
@sjmiller7 does this still happen after #1169? I'd test it myself to find out, but I don't have the AWS config stuff locally

**Comment by colbygallup** [comment]
I tested this using featurepeek on another PR, and it looks like this behavior no longer occurs.
